From 8a27264570a529a6830f83a80aa69f30e8dd8408 Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <josef.schlehofer@nic.cz>
Date: Fri, 25 Dec 2020 13:51:54 +0100
Subject: [PATCH] reforis: add HOST_PYTHON3_PACKAGE_BUILD_DEPENDS

This was newly introduced in OpenWrt master, so let's use it as well.
Otherwise, we don't have reForis in HBD branch
---
 web/reforis/reforis/Makefile | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/web/reforis/reforis/Makefile b/web/reforis/reforis/Makefile
index e72119273..a9136d4e7 100644
--- a/web/reforis/reforis/Makefile
+++ b/web/reforis/reforis/Makefile
@@ -20,9 +20,8 @@ PKG_MAINTAINER:=CZ.NIC <packaging@turris.cz>
 PKG_LICENSE:=GPL-3.0-or-later
 PKG_LICENSE_FILES:=LICENSE
 
-PKG_BUILD_DEPENDS:=\
-	node/host \
-	reforis-distutils/host \
+HOST_PYTHON3_PACKAGE_BUILD_DEPENDS:=reforis-distutils
+PKG_BUILD_DEPENDS:=node/host
 
 include $(INCLUDE_DIR)/package.mk
 include $(TOPDIR)/feeds/packages/lang/python/python3-package.mk
-- 
2.25.1

